### PR TITLE
feat: changing grids in this feature

### PR DESCRIPTION
**Pull request by mariyasada**
one more grid system was added which contains three items in the grid
grid

### CODE REVIEW

File: stylesheets/modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+modalCodeBtn.addEventListener("click", () => {
+        console.log("clicked");
+        modalCodeDiv.style.display == "block"
+          ? (modalCodeDiv.style.display = "none")
+          : (modalCodeDiv.style.display = "block");
+      });
+
+// const modalCodeBtn = document.querySelector("#modal-code-btn");
```
Comment (shaksham08) [inline stylesheets/modal/modal.js]: You can remove commented code

**Review by shaksham08 — APPROVED** [review]
Code looks clean just added one comment.

File: stylesheets/modal/modal.css [hunk 2]
```diff
@@ -1,33 +1,54 @@
...
-    display: flex;
-    align-items: center;
-    justify-content: flex-end;
-    gap: 2rem;
-    padding: 2rem;
-    width: 100%;
-}
\ No newline at end of file
+  display: flex;
+  align-items: center;
```
Comment (ozairkhan2810) [inline stylesheets/modal/modal.css:30]: can combine display:flex and align-items:center with other 2 classes to remove redundancy

**Review by ozairkhan2810 — APPROVED** [review]
just one suggestion, rest looks good

**Pull request merged**